src/system/mod.rs

```diff
@@ -123,17 +123,17 @@ impl System {
         &self,
         origin: TrackingUniverseOrigin,
     ) -> Option<(EventInfo, TrackedDevicePose)> {
-        let mut event = unsafe { mem::uninitialized() };
-        let mut pose = unsafe { mem::uninitialized() };
+        let mut event = unsafe { mem::MaybeUninit::uninit() };
+        let mut pose = unsafe { mem::MaybeUninit::uninit() };
```

Comment:
The `unsafe` can be removed here; constructing a `MaybeUninit` is always safe.

Reply:
Ah, got it! I've pushed the change.